java/data-management/strings-ii/using-printstream-format-to-format-a-string-before-printing.md

```diff
@@ -21,7 +21,7 @@ links:
 int i = 500;
 float f = 2.36;
 String s = "Java";
-//System.out is Prtinstream object
+//System.out is Printstream object
```

Comment:
we should add a space after comment characters (`//`) and I think the name is `PrintStream` ([source](https://docs.oracle.com/javase/7/docs/api/java/io/PrintStream.html))

```suggestion
// System.out is a PrintStream object
```